tools/spack/cp2k-dependencies.yaml

```diff
@@ -9,7 +9,7 @@ spack:
     - "libxsmm @1.17"
     - "fftw @3.3.10 +openmp"
     - "libxc @6.2.2"
-    - "spglib @1.16.2"
+    - "spglib @2.3.1"
```

Comment:
```suggestion
    - "spglib @2.3.0"
```

@LecrisUT, I haven't yet added `2.3.1` in Spack (will do later today), please use `2.3.0`.

Reply:
Still the same issue though :shrug:

Reply:
Actually, `2.3.0` and `2.2.0` might be too new. I need to fix #3246 before we can update the Spack version (and have the new `spglib` versions available).

Reply:
For reference: https://github.com/spack/spack/pull/43345